manifests/c/CodecGuide/K-LiteCodecPack/Mega/17.3.0/CodecGuide.K-LiteCodecPack.Mega.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CodecGuide.K-LiteCodecPack.Mega
+PackageVersion: 17.3.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files3.codecguide.com/K-Lite_Codec_Pack_1730_Mega.exe
+  InstallerSha256: 6DFE2A03FC8B03A3C494A84EF6E621D5C977D415CE7128957720849AF29B9DF5
+  AppsAndFeaturesEntries:
+  - DisplayName: K-Lite Mega Codec Pack 17.2.5

```

```suggestion
  - DisplayName: K-Lite Mega Codec Pack 17.3.0
```
This must match Control Panel